lib/logstash/inputs/syslog.rb

```diff
@@ -118,6 +119,7 @@ def server(protocol, output_queue)
   rescue => e
     if !stop?
       @logger.warn("syslog listener died", :protocol => protocol, :address => "#{@host}:#{@port}", :exception => e, :backtrace => e.backtrace)
+      @metric_errors.increment(:listener_exception)
```

Comment:
@jordansissel as of now the output will be `{ "errors" { "listener_exception" => 123 } }`, but I do like also the idea to remove the `exception` part, lets find consensus with @ph proposals in other PR.